database/migrations/20190723132825-create-comments.js

```diff
@@ -34,6 +34,9 @@ module.exports = {
           key: 'id',
         },
       },
+      articleSlug: {
```

Comment:
Can an article's slug be empty?

Reply:
Article slug isn't really necessary on this table since there is also an articleId. I just thought I should add it in the comments table just in case I need a use for it.